### ISSUE TITLE
y scale

### ISSUE THREAD

**Issue by coot** [hdgarrood/purescript-benchotron-svg-renderer#8]
As you can see on the screenshot, there is `0.01` and `1.00e-2` while it should be `1.00e-3`, and also the last `0.06` is twice.
![screenshot from 2017-07-29 08-07-41](https://user-images.githubusercontent.com/698209/28742624-314f8c74-7435-11e7-9313-387264b1b09d.png)

**Comment by hdgarrood** [comment]
Yeah, I've been wanting to fix this for a while. Thanks for the report!
